### ISSUE TITLE
Modeltimeh2o does not return the h2o model class?  

### ISSUE THREAD

**Issue by twoolston** [business-science/modeltime.h2o#29]
Matt love the package.   But is there a way to extract the the h2o model class from the model_fitted?   So the h2o package functions can use the object?

**Comment by mdancho84** [comment]
Yes absolutely. The underlying H2O model is stored as part of the file.

**Comment by mdancho84** [comment]
Here's the save and load code. https://github.com/business-science/modeltime.h2o/blob/master/R/core-save_and_load.R

**Comment by twoolston** [comment]
Thanks!  Love your work.

**Comment by mdancho84** [comment]
Awesome! I appreciate it.
